web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -52,6 +54,13 @@ const BookingModalForm = props => {
     }
   };
 
+  const remainingDays = (getTotalDays - getApprovedDays());
+  const composeRemainingHolsErrorMessage = () => {
+    if (remainingDays <= 0 || bookingDuration > remainingDays) { 
+      return true;
```

Comment:
I think this could be named better as there's no composition going on in the function, and we can get rid of the `if` as well

```
const showRemainingHolsErrorMessage = () => {
  return (remainingDays <= 0 || bookingDuration > remainingDays)
}
```